tools/genevergreen/generate/generate.go

```diff
@@ -70,11 +70,19 @@ var (
 	}
 )
 
-func newDependency(toolName, os, serverVersion, repo string) shrub.TaskDependency {
-	return shrub.TaskDependency{
-		Name:    fmt.Sprintf("push_%s_%s_%s_stable", toolName, newOs[os], repo),
-		Variant: fmt.Sprintf("release_%s_publish_%s", toolName, strings.ReplaceAll(serverVersion, ".", "")),
+func newDependency(toolName, os, serverVersion, repo string) []shrub.TaskDependency {
+	var deps []shrub.TaskDependency
+	architectures := Distros[newOs[os]].architectures
+
+	for _, arch := range architectures {
+		dep := shrub.TaskDependency{
+			Name:    fmt.Sprintf("push_%s_%s_%s_%s_%s_stable", toolName, newOs[os], repo, arch, strings.ReplaceAll(serverVersion, ".", "")),
+			Variant: fmt.Sprintf("generated_release_%s_publish_%s", toolName, strings.ReplaceAll(serverVersion, ".", "")),
+		}
+		deps = append(deps, dep)
```

Comment:
from `push_atlascli_amazon2_enterprise_stable` to `push_atlascli_amazon2_org_aarch64_42_stable`